.travis.yml

```diff
@@ -5,9 +5,18 @@ python:
     - 3.4
     - 3.5
     - 3.6
+matrix:
+    include:
+        - python: "3.7"
+          dist: xenial
+          sudo: true
 install:
+    - pip install coveralls coverage
     - pip install tox-travis
+    - pip install -U -e .[test]
 script:
-    - tox
+    - coverage run -m zope.testrunner --test-path=src
```

Comment:
There's no '.coveragerc' in this repo, so the coverage reports are probably going to be very off. https://coveralls.io/github/zopefoundation/zope.globalrequest

Reply:
That's odd, I committed it, git status isn't listing it and it didn't show up. I'll fix that.

Reply:
nevermind, i oversaw the .gitignore settings.